eng/pipelines/dataplane-pr.yml

```diff
@@ -0,0 +1,90 @@
+trigger:
+  branches:
+    include:
+    - feature/*
+pr:
+  branches:
+    include:
+    - feature/*
+    - '*-preview'
+  paths:
+    include:
+    - eng/pipelines/mgmt-pr.yml
+    - sdk/applicationinsights/applicationinsights-query
```

Comment:
How did you generate this list? Should we consider a script that refreshes them as needed like we have in other repos or is the set pretty static in this case.

Reply:
This list is static and I have selected it from previous management pipeline yamls without arm paths. @ramya-rao-a has  shared a static list to which I double checked these paths.
https://github.com/Azure/azure-sdk-for-js/issues/16663

Reply:
yup, all looks good here